spec/requests/comments_spec.rb

```diff
@@ -332,6 +332,24 @@
 
   describe "PATCH /comments/:comment_id/hide" do
     include_examples "PATCH /comments/:comment_id/hide or unhide", path: "hide", hidden: "true"
+
+    context "with notifications" do
+      let(:commentable_author) { create(:user) }
+      let(:article) { create(:article, user: commentable_author) }
+      let(:parent_comment) { create(:comment, commentable: article, user: commentable_author) }
+      let(:notification) do
+        Notification.send_new_comment_notifications_without_delay(parent_comment)
+      end
```

Comment:
```ruby
     Notification.send_new_comment_notifications_without_delay(parent_comment)
```
 Always seems to return nil hence this test passes always. I think I might need help to understand this.

Reply:
Hi @Rafi993, that method doesn't return the notification object, you can get it by fetching it from the user's notification:

1. create comment
1. create notification
1. fetch the notification from the user's `.notifications` relation
1. hide comment
1. check the notification doesn't exist in anymore (eg. `Notification.exists?(id: notification.id)` should fail)

Reply:
> 5\. Notification.exists?(id: notification.id)

Thanks @rhymes I have fixed the test now. I'll squash my commits too.

Reply:
@Rafi993 thanks! ps. no need to squash them in the future as we squash when we merge ;)